src/ui/public/vis/response_handlers/vislib.js

```diff
@@ -78,18 +78,14 @@ const handlerFunction =  function (convertTable) {
   };
 };
 
-const VislibSeriesResponseHandlerProvider = function (Private) {
-  const buildPointSeriesData = Private(AggResponsePointSeriesProvider);
-
+const VislibSeriesResponseHandlerProvider = function () {
```

Comment:
nit: If we're cleaning this up to remove `Private`, should we rename to simply `VislibSeriesResponseHandler`, since it is no longer a "Provider" ?

Reply:
we still need it like this for our response handler registry. that will need to be cleaned up as well (next pr)